src/components/Card/DetailCard.tsx

```diff
@@ -0,0 +1,71 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import React from 'react';
+import { Tags } from '../Elements/Tag';
+import styles from './DetailCard.module.scss';
+import { Price, Tag } from 'api/@types';
+
+export type DetailCardProps = {
+  content: string;
+  thumbnail: string;
+  link: string;
+  tags: Tag[];
+  price: Price;
+};
+
+export const DetailCard: React.FC<DetailCardProps> = ({
+  content,
+  thumbnail,
+  link,
+  tags,
+  price,
+}) => {
+  return (
+    <Link href={link}>
+      <div className={styles.wrapper}>
+        <figure className={styles.thumbnail_wrapper}>
+          <Image
+            src={thumbnail}
+            alt={content}
+            fill
+            className={styles.thumbnail}
+          />
+          {price.discounted && (
+            <div className={styles.thumbnail__sale}>
+              {Math.ceil(price.discount_percentage)}%OFF
+            </div>
```

Comment:
[NITS]

figure要素内のキャプションはfigcaptionを使用するのが良さそうです！（他のコンポーネントも同様）
https://developer.mozilla.org/ja/docs/Web/HTML/Element/figure
```suggestion
            <figcaption className={styles.thumbnail__sale}>
              {Math.ceil(price.discount_percentage)}%OFF
            </figcaption>
```

Reply:
なるほどです、figcaptionをつけるという観点は抜けていました。ありがとうございます

しかし、キャプションは`画像"そのもの"`の説明につけるものであり、例えばソファー商品があったとして、そのソファーの画像に対する写真として「ソファーの画像」などのキャプションがつくのではないかと思います。
今回のケースでは、`商品"そのもの"`が「n％OFF」という話なので、figcaptionの想定の使い方ではないような気がしました！

Reply:
おっしゃる通りですね🙇‍♂️
figure要素使っているのでfigcaption使わないと。。。という縛りみたいなものが自分の中にありました。。。🙇‍♂️